cmd/osbuild-composer/composer.go

```diff
@@ -218,7 +222,7 @@ func (c *Composer) Start() error {
 		logrus.Fatal("neither the weldr API socket nor the composer API socket is enabled, osbuild-composer is useless without one of these APIs enabled")
 	}
 
-	var localWorkerAPI, remoteWorkerAPI, composerAPI *http.Server
```

Comment:
Nitpick: sorry for not noticing earlier, but it would be nice to be consistent and use `API` suffix in the `*http.Server` variable name, as others do. But I'm OK with keeping it as is.